BUILD.gn

```diff
@@ -4,15 +4,27 @@
 
 group("default") {
   testonly = true
-  deps = [
-    "//flutter",
-  ]
+  if (target_os == "wasm") {
+    deps = [
+      "//flutter/web_sdk/wasm:wasm_artifacts"
```

Comment:
I think if we change things here, we should just make some specific target for `wasm` e.g. `//flutter/wasm` that is not specific to the web_sdk.

Alternatively, we should do this in `//flutter/BUILD.gn`.

Reply:
Putting it in `//flutter/BUILD.gn` causes problems due to targets and imports eventually leading somewhere that will throw due to `target_cpu = "wasm"`. But putting it in `//flutter/wasm/BUILD.gn` sounds good to me. The WASM stuff needs to be quarantined from the normal Flutter SDK build rules because otherwise `gn gen` will error out.